storage/backup/backup-cli/src/backup_types/transaction/manifest.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::storage::FileHandle;
+use anyhow::{ensure, Result};
+use libra_types::transaction::Version;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+pub struct TransactionChunk {
```

Comment:
comment? right-inclusive?

Reply:
I think the "last_xx" naming is pretty obvious inclusive.. but sure..